_hooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+
+# A git hook script that checks for trailing whitespace
```

Comment:
Can we structure this the same as bigsanity (same example command in README, similar wording)

https://github.com/m-lab/bigsanity/tree/master/hooks